contrib/gocql/gocql/gocql.go

```diff
@@ -111,6 +111,18 @@ func (tq *Query) newChildSpan(ctx context.Context) ddtrace.Span {
 }
 
 func (tq *Query) finishSpan(span ddtrace.Span, err error) {
+	// ErrNotFound is technically an error from gocql point of view.
+	//
+	// However, it is returned when "no data is available", even if
+	// the request was perfectly correct and executed well. It is
+	// only a marker of "zero rows returned".
+	//
+	// So APM graphs/data can expose a very high error rate if queries
+	// happen to try and select data which is just not there.
+	if err == gocql.ErrNotFound {
+		err = nil
```

Comment:
I think this wrapper must not alter the return values returned to the caller. Only alter what is sent to trace spans.